react/orders101/src/features/cart/Cart.jsx

```diff
@@ -0,0 +1,36 @@
+import { useContext } from "react";
+import BasketClose from "../BasketClose";
+import CartBox from "./CartBox";
+import Checkout from "./Checkout";
+import { ShoppingCartContext } from "../../App";
+
+function Cart() {
+  const { cartItems } = useContext(ShoppingCartContext);
+
+  return (
+    <div class="cart">
```

Comment:
As discussed today and related to [this comment](https://github.com/MinhazH99/orders101/pull/62/files#r1576089671), we could set the visibility dynamically with something like

```
<div class="cart" style={{display: isOpen ? 'block' : 'none'}}>
```

What do you think?

Reply:
Thanks Sara, code above has been implemented